src/fswAlgorithms/attControl/thrMomentumManagementCpp/_UnitTest/test_thrMomentumManagementCpp.py

```diff
@@ -0,0 +1,106 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2024, Laboratory of Atmospheric and Space Physics, University of Colorado at Boulder
+#
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+
+import inspect
+import os
+
+import pytest
+import numpy as np
+
+filename = inspect.getframeinfo(inspect.currentframe()).filename
+path = os.path.dirname(os.path.abspath(filename))
+
+# Import all of the modules that we are going to be called in this simulation
+from Basilisk.utilities import SimulationBaseClass, macros, fswSetupRW
+from Basilisk.fswAlgorithms import thrMomentumManagementCpp
+from Basilisk.architecture import messaging
+
+
+@pytest.mark.parametrize("hsMinCheck", [(0), (1)])
+def test_thrMomentumManagement(show_plots, hsMinCheck):
+    unitTaskName = "unitTask"               # arbitrary name (don't change)
```

Comment:
:large_blue_circle: There are many redundant comments throughout the unit tests in the code base. We should seek to clean them out as we go.

Reply:
Done.